src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/auditing/AuditEvent.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.auditing
+
+import java.time.LocalDateTime
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.Id
+import javax.persistence.Table
+import javax.validation.constraints.NotBlank
+
+@Entity
+@Table(name = "AUDIT_EVENTS")
+data class AuditEvent(
+  @Enumerated(EnumType.STRING)
+  @Column(name = "event_type", nullable = false)
+  var eventType: AuditEventType,
+
+  @Column(name = "created_at", nullable = false)
+  val createdAt: LocalDateTime,
+
+  @Column(name = "username", nullable = false)
+  @get: NotBlank(message = "User cannot be blank")
+  val username: String,
+
+  @Column(name = "details", nullable = false)
```

Comment:
Probably worth specifying a length on this e.g:

`  @Column(name = "details", nullable = false, length = 500)
`
Not sure if 500 is too smallb ut the default of 255 will probably be too small.